src/page-multi-facility/MultiFacilityImpactDashboard.tsx

```diff
@@ -20,15 +20,32 @@ const MultiFacilityImpactDashboardContainer = styled.main.attrs({
   `,
 })``;
 
-const MultiFacilityImpactDashboard: React.FC = () => {
+interface MultiFacilityImpactDashboardType = {
+  baselineScenario?: object;
+}
+
+const MultiFacilityImpactDashboard<MultiFacilityImpactDashboardType>: React.FC = ({ baselineScenario }) => {
   const { data: localeDataSource } = useLocaleDataState();
 
   const [facilities, setFacilities] = useState({
     data: [] as Facilities,
     loading: true,
   });
 
+  const [scenario, setScenario] = useState({
+    data: {},
+    loading: true
+  })
+
   useEffect(() => {
```

Comment:
I don't know enough about React, but is each piece of managed state supposed to get its own `useEffect` or is it common to nest multiple calls under a single `useEffect` like we're doing here?

Reply:
I think the way to think of it is as discrete operations. If you have two things you want to happen independently of each other, they should be their own hooks. If you have two things that you want to happen in tandem or in a particular order, they may live in the same hook.

Reply:
In this case, I think maybe the thinking was that loading one or the other but not both might be a poor experience so we should place it within a single hook?

Reply:
Ah okay--I actually just updated it to be 2 hooks before I saw this comment. I think will mean the facilities fetching will not depend on the scenario fetching to complete first, so I think they would happen independently from each other. If we want to ensure both are loaded before loading the content maybe that would be more of a conditional rendering?

Reply:
I think it's likely a non-issue as currently constructed, but it should become apparent quickly in final testing of so. Let's leave it as is!